tests/test_model.py

```diff
@@ -345,7 +346,87 @@ def test_remove_unused_symbols(self):
             self.assertEqual(num_removed, 1)
             self.assertEqual(model.num_symbols(), 9)
 
-    def test_serialization(self):
+    def test_state_size(self):
```

Comment:
Git got confused about which tests were changing. These are all unchanged, I simply moved the serialization tests to a different test class, and added one more test. So should be no functional change.